inc/vite.php

```
@@ -93,6 +91,16 @@ private function load_manifest() {
         }
     }
 
+    /**
+     * Get manifest data, lazy-loading if necessary
+     */
+    private function get_manifest() {
+        if (empty($this->manifest)) {
+            $this->load_manifest();
+        }
+        return $this->manifest;
+    }
```

PHPMD (2.15.0)

[error] The method get_manifest is not named in camelCase. (undefined)

(CamelCaseMethodName)